elks/arch/i86/drivers/block/ata-cf.c

```diff
@@ -0,0 +1,206 @@
+/**********************************************************************
+ * ELKS ATA-CF driver
+ *
+ * Please note that this driver does not currently support partitions.
+ *
+ * This driver is largely based on Greg Haerr's SSD driver.
+ *
+ * Ferry Hendrikx, June 2025
+ **********************************************************************/
+
+#include <linuxmt/config.h>
+#include <linuxmt/kernel.h>
+#include <linuxmt/errno.h>
+#include <linuxmt/debug.h>
+#include <arch/ata.h>
+#include "ata.h"
+#include "ata-cf.h"
+#include "blk.h"
+
+
+/**********************************************************************
+ * device initialisation
+ **********************************************************************/
+
+jiff_t ata_cf_timeout;
+
+char ata_cf_initialized;
+sector_t ata_cf_num_sects[NUM_DRIVES];     /* max # sectors on SSD device */
```

Comment:
These three variable declarations aren't used in any other file, right? Can you please make them static? Thanks.